lambdas/test/apis/profiles/EnsOwnership.spec.ts

```diff
@@ -50,22 +56,22 @@ describe('Ensure ENS filtering work as expected', () => {
     const originalAddress = '0x079BED9C31CB772c4C156F86E1CFf15bf751ADd0'
     const ensOwnership = buildOwnership(fetcher)
 
-    await ensOwnership.areNamesOwned(originalAddress, ['marcosnc', 'invalid_name'])
-    await ensOwnership.areNamesOwned(originalAddress, ['marcosnc', 'invalid_name'])
+    await ensOwnership.areNamesOwnedByAddress(originalAddress, ['marcosnc', 'invalid_name'])
+    await ensOwnership.areNamesOwnedByAddress(originalAddress, ['marcosnc', 'invalid_name'])
 
     verify(mockedFetcher.queryGraph(anything(), anything(), anything())).once()
   }, 100000)
 
-  it(`When getting the owned names for a different entity, then the graph is consulted twice`, async () => {
+  it(`When getting the owned names for a different entity, then the graph is consulted once`, async () => {
     const mockedFetcher: Fetcher = getMockedFetcher()
     const fetcher: Fetcher = instance(mockedFetcher)
     const originalAddress = '0x079BED9C31CB772c4C156F86E1CFf15bf751ADd0'
     const ensOwnership = buildOwnership(fetcher)
 
-    await ensOwnership.areNamesOwned(originalAddress, ['marcosnc', 'invalid_name'])
-    await ensOwnership.areNamesOwned('anotherAddress', ['marcosnc'])
+    await ensOwnership.areNamesOwnedByAddress(originalAddress, ['marcosnc', 'invalid_name'])
+    await ensOwnership.areNamesOwnedByAddress('anotherAddress', ['marcosnc'])
 
-    verify(mockedFetcher.queryGraph(anything(), anything(), anything())).times(2)
+    verify(mockedFetcher.queryGraph(anything(), anything(), anything())).once()
```

Comment:
With the `Cache<Name, owner: Address | null>` approach, we don't have to check the graph twice! 🎉